test/routes.post.test.js

```diff
@@ -0,0 +1,74 @@
+const request = require('supertest');
+const app = require('../src/backend/web/app');
+const { addPost } = require('../src/backend/utils/storage');
```

Comment:
Instead of using `storage` and `addPost` directly, please use `post.js`, `new Post(...)` and `post.save()`.